docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -21,168 +21,109 @@ helpviewer_keywords:
-
-- \* (use this value to specify all symbols)
-- namespace
-- class
-- struct
-- interface
-- enum
-- property
-- method
-- field
-- event
-- delegate
-- parameter
-- type_parameter
-- local
-- local_function
-
-> [!NOTE]
-> Tuple members aren't currently supported.
-
-### Accessibility levels of symbols
-
-To describe the accessibility levels of the symbols you want the naming rule to apply to, specify a property name in the following format:
-
-`dotnet_naming_symbols.<symbolTitle>.applicable_accessibilities = <values>`
-
-The following list shows the allowable values, and you can specify multiple values by separating them with a comma.
-
-- \* (use this value to specify all accessibility levels)
-- public
-- internal or friend
-- private
-- protected
-- protected\_internal or protected_friend
-- private\_protected
-- local
-
-   The `local` accessibility level applies to symbols defined within a method. It's useful for defining naming rules for symbols whose accessibility can't be specified in code. For example, if you specify `applicable_accessibilities = local` on a naming convention for constants (`required_modifiers = const`), the rule applies only to constants defined within a method and not those defined in a type.
-
-   ```csharp
-   class TypeName
-   {
-     // Constant defined in a type.
-     const int X = 3;
-
-     void Method()
-     {
-       // Constant defined in a method with "local" accessibility.
-       const int Y = 4;
-     }
-   }
-   ```
-
-### Symbol modifiers (optional)
-
-To describe the modifiers of the symbols you want the naming rule to apply to, specify a property name in the following format:
+Each property should only be set once, but some settings allow multiple comma-separated values.
 
-`dotnet_naming_symbols.<symbolTitle>.required_modifiers = <values>`
+The order of the properties is not important.
 
-The following list shows the allowable values (separate multiple values with a comma):
+### \<prefix>
 
-- `abstract` or `must_inherit`
-- `async`
-- `const`
-- `readonly`
-- `static` or `shared`
+**\<prefix>** specifies which kind of element is being defined -- naming rule, symbol group, or naming style -- and must be one of the following:
 
-   > [!NOTE]
-   > If you have a naming rule for `static` or `shared` symbols, it is also applied to `const` symbols because they are implicitly static. If you don't want the `static` naming rule to apply to `const` symbols, create a separate naming rule for `const` symbols.
+| To set a property for | Use the prefix | Example |
+| --- | --- | -- |
+| Symbol group | `dotnet_naming_symbols` | `dotnet_naming_symbols.interface.applicable_kinds = interface` |
+| Naming style | `dotnet_naming_style` | `dotnet_naming_style.pascal_case.capitalization = pascal_case` |
+| Naming rule | `dotnet_naming_rule` | `dotnet_naming_rule.types_should_be_pascal_case.severity = suggestion`
```

Comment:
Can you move this up to be the first row in the table, to match the order you present the properties on line 44.